lib/memcached/client.rb

```diff
@@ -10,17 +10,7 @@ class Client
     FLAGS = 0x0
 
     DEFAULTS = {
-      :hash => :fnv1_32,
-      :distribution => :consistent_ketama,
-      :ketama_weighted => true,
-      :retry_timeout => 60,
-      :timeout => 0.25,
-#      :poll_max_retries => 1, TODO: doesnt exist anymore
-      :connect_timeout => 0.25,
-      :hash_with_prefix_key => true,
-      :server_failure_limit => 2,
       :verify_key => true,
-      :io_key_prefetch => true,
     }
```

Comment:
Gotcha. I've brought back some of the defaults which are sensible (hashing mode and so on). AFAICT, the rest of the defaults (timeouts and so on) are *not very sensible* and it's better to use the ones that `libmemcached` suggests.